src/meta/processors/index/GetTagIndexProcessor.cpp

```diff
@@ -42,7 +42,7 @@ void GetTagIndexProcessor::process(const cpp2::GetTagIndexReq& req) {
 
   auto item = MetaKeyUtils::parseIndex(nebula::value(indexItemRet));
   if (item.get_schema_id().getType() != nebula::cpp2::SchemaID::Type::tag_id) {
-    LOG(INFO) << "Get Tag Index Failed: Index Name " << indexName << " is not TagIndex";
+    VLOG(2) << "Get Tag Index Failed: Index Name " << indexName << " is not TagIndex";
```

Comment:
I don't understand, is this an error log?

Reply:
look likes it an error indeed.